sled-agent/src/bootstrap/http_entrypoints.rs

```diff
@@ -106,8 +108,10 @@ impl BootstrapAgentApi for BootstrapAgentImpl {
         rqctx: RequestContext<Self::Context>,
         body: TypedBody<RackInitializeRequest>,
     ) -> Result<HttpResponseOk<RackInitUuid>, HttpError> {
+        const SKIP_TIMESYNC: bool = false;
```

Comment:
Just checking my understanding - in this path, we're getting an RSS config from an external client, which means we expect all the sled-agents were configured with `skip-timesync = false`, because skipping timesync is only applicable to single-sled deployments where the RSS config comes in as a file?

This seems a little fragile but isn't anything new in this PR. Maybe just worth a comment explaining what we believe the situation to be, in this case?

Reply:
That's right (and you're right about the fragility; I'm not sure how to plumb that through, however). Comment added.